src/maintainer/conda_forge_yml.rst

```diff
@@ -79,6 +80,34 @@ mapping for Azure-specific configuration options. For example:
       # built packages) as an Azure pipeline artifact that can be downloaded
       store_build_artifacts: False
 
+.. _self-hosted_azure-config:
+
+self-hosted azure
+-----------------
+Conda-Smithy is capable of doing things beyond CF's purposes, it also allows setting up
+self-hosted agents for non-CF infrastructures. Here is an example demonstrating that:
```

Comment:
A general intro like this should go to [here](https://conda-forge.org/docs/maintainer/infrastructure.html?highlight=smith) as @viniciusdc suggested in https://github.com/conda-forge/conda-forge.github.io/issues/1337#issuecomment-814997213. Let us just focus on what's needed to set it up in this file, something like:
```suggestion
Below is the configuration for setting up a self-hosted Azure agent:
```

Reply:
>A general intro like this should go to here as @viniciusdc suggested in #1337 (comment). Let us just focus on what's needed to set it up in this file, something like:


Shall  I add all these points under the main "Smithy" heading or make a new heading. (although in my view it will be better to add all these points under the main "Smithy" heading after this line `conda-smithy also contains the command line tool that you should use if you rerender manually from the command line (see Rerendering feedstocks
`
Let me know your views on this, and I will add those points as you suggest

Reply:
Sounds reasonable to me 🙂